lambdas/webhook_processor/webhook_processor.py

```diff
@@ -127,6 +179,224 @@ def _process_results_for_failures(results):
     return results_count
 
 
+def _parse_hook_for_testing_info(payload, event_type):
+    """
+    Parse the webhook to find the commit sha,
+    as well as the arguments needed to find the
+    jobs_list.
+    Returns:
+        Tuple with commit sha and list of jobs that
+        should be triggered. If the event type is not
+        pull_request, return empty values
+    """
+    ignore = False
+
+    if event_type == 'pull_request':
+        # find the repo and whether the pr is being merged
+        repository = payload['pull_request']['base']['repo']['name']
+        is_merge = payload['pull_request']['merged']
+
+        # check if the base branch is an openedx release branch
+        base_ref = payload['pull_request']['base']['ref']
+        if base_ref == EUCALYTPUS_BRANCH:
+            target = "eucalyptus"
+        elif base_ref == FICUS_BRANCH:
+            target = "ficus"
+        else:
+            target = "master"
+
+        if payload['action'] == 'closed':
+            sha = payload['pull_request']['merge_commit_sha']
+            if not is_merge:
+                # if the PR was closed and not merged,
+                # no jobs will be triggered
+                ignore = True
+        else:
+            # PR was either opened or "synchronized"
```

Comment:
Pls add a note explaining that "synchronized" is the action name for pushing a new commit (? if that's true. or if not whatever the explanation really is.)